hazelcast/src/main/java/com/hazelcast/map/impl/operation/ConsumeAccumulatorOperation.java

```diff
@@ -46,15 +47,16 @@
  * Processes remaining items in {@link Accumulator} instances on a partition.
  * If the partition is not owned by this node, {@link Accumulator} will be removed.
  */
-public class AccumulatorConsumerOperation extends Operation implements PartitionAwareOperation, IdentifiedDataSerializable {
+public class ConsumeAccumulatorOperation extends Operation
+        implements PartitionAwareOperation, MutatingOperation, IdentifiedDataSerializable {
```

Comment:
Same here. It seems this is some background periodic task. If this is the case, we shouldn't prevent it from being run if quorum is not present.

Reply:
fixed